src/averageAges.js

```diff
@@ -37,7 +37,19 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const female = people
+    .filter(person => person.sex.toLowerCase() === 'f'
+      && (!withChildren
+        || people.some(child => child.mother === person.name)));
+
+  return avarageAge(female);
+}
+
+function avarageAge(person) {
```

Comment:
```suggestion
function getAverageAge(people) {
```
Get used to name functions according to these rules:
![image](https://user-images.githubusercontent.com/96409563/228935897-1502b256-12de-45f5-a0a8-e60deebb21e8.png)
Also, function receives an array of people, not one person